src/interface/definitions.yaml

```diff
@@ -144,13 +144,33 @@ ThirdPartyAuthorizationRequest:
   title: ThirdPartyAuthorizationRequest
   type: object
   description: The Request object for verifying an authorization
-  # TODO: finish me!!!
   properties:
     challenge: 
       type: string
-      description: The original Challenge Object as a JSON string
+      description: The original Challenge (Quote response) object as a JSON string
+    value:
+      type: string
+      description: The signed Challenge (quote response) object as base64 encoded string 
+    consentId:
+      - $ref: '#/definitions/CorrelationId'
```

Comment:
Forgive my ignorance -- what's a correlation ID? Is this as intended? Why does `sourceAccountId` point to an AccountId (see below), but this doesn't point to a ConsentId ?

Reply:
`Correlation ID` is how the current OpenAPI definition defines the ID for any resource such as `Consent`.
![Screenshot 2020-07-27 at 2 09 29 PM](https://user-images.githubusercontent.com/41443552/88521707-e77dee00-d012-11ea-8bde-c6e1f51dc116.png)

Reply:
There is no standalone definition of a `ConsentId`. It is just a part of the `Consent` defintion.

Reply:
'sourceAccountId` points to an `AccountId' because `AccountId` is a standalone object in our OpenAPI spec.

Reply:
We may have to wait for @lewisdaly or someone to chime in on this, but I believe that we should have a ConsentId type, and Consent.id should reference that type (after all, it's a proper resource that's part of Mojaloop, so why wouldn't it get its own ID type?)

Reply:
@jgeewax we don't need `ConsentId` type. In `mojaloop` we are using CorrelationId type for resource ids. The resource Ids are used also as URL parameters, so there must be one standard for all - it is UUIDv4 format if I remember correctly - and there is proper regexp for this. Accounts are not resources in `mojaloop` context, but only the reference id to be presented to End User and be a part of Consent. We created AccountId which is defined as a simple `string` without `regexp`, to have one reusable definition to easily maintain  OpenApi spec. We can have`ConsentId` type and it should always resolve to CorrelationId elsewhere we will brake `mojaloop` convention for resource ids.